package.json

```diff
@@ -20,6 +20,9 @@
     },
     "./rules": {
       "types": "./lib/types/rules/index.d.ts"
+    },
+    "./universal": {
+        "default": "./lib/linter/linter.js"
```

Comment:
This would also export [`getLinterInternalSlots`](https://github.com/eslint/eslint/blob/59cfc0f1b3bbb62260602579f79bd1c36ab5a00f/lib/linter/linter.js#L2369C5-L2369C27). Can we add a new module `universal.js` (like [`api.js`](https://github.com/eslint/eslint/blob/main/lib/api.js)) and export `{ Linter }` from it?